src/Riok.Mapperly/Descriptors/Mappings/TypeMappingBuildContext.cs

```diff
@@ -55,10 +55,13 @@ SyntaxFactoryHelper syntaxFactory
     /// </summary>
     /// <param name="sourceBuilder">A function to build the source access for the new context.</param>
     /// <returns>The new context and the scoped name of the source.</returns>
-    public (TypeMappingBuildContext Context, string SourceName) WithNewScopedSource(Func<string, ExpressionSyntax> sourceBuilder)
+    public (TypeMappingBuildContext Context, string SourceName) WithNewScopedSource(
+        Func<string, ExpressionSyntax> sourceBuilder,
+        string sourceName = DefaultSourceName
```

Comment:
IMO adding this default parameter to the currently parameterless overload would simplify the caller side.